### PR TITLE
BACKLOG-10130 add displayName for the form field

### PR DESCRIPTION
**Pull request by Taybou**

### CODE REVIEW

File: src/main/java/org/jahia/modules/contenteditor/api/forms/EditorFormField.java [hunk 1]
```diff
@@ -58,32 +62,37 @@ public EditorFormField(String name,
...
+            field.mandatory,
+            field.valueConstraints == null ? null : field.valueConstraints.stream()
+                .map(constraint -> new EditorFormFieldValueConstraint(constraint))
+                .collect(Collectors.toList()),
+            field.defaultValues == null ? null : new ArrayList<>(field.defaultValues),
+            field.removed,
+            field.targets == null ? null : field.targets.stream()
+                .map(target -> new EditorFormFieldTarget(target))
+                .collect(Collectors.toList()),
+            // FIXME no deep copy here
```
Comment (FBerthelot) [inline src/main/java/org/jahia/modules/contenteditor/api/forms/EditorFormField.java]: Why not Fix this in the PR ?

Comment (Taybou) [reply src/main/java/org/jahia/modules/contenteditor/api/forms/EditorFormField.java]: Not related to this task. We should create dedicated task to fix this one.

Comment (FBerthelot) [reply src/main/java/org/jahia/modules/contenteditor/api/forms/EditorFormField.java]: The task is created ?

If not, I think you should create it :)

Comment (Taybou) [reply src/main/java/org/jahia/modules/contenteditor/api/forms/EditorFormField.java]: Ok, I'll created.

Comment (Taybou) [reply src/main/java/org/jahia/modules/contenteditor/api/forms/EditorFormField.java]: After Discussion with @cedricmoitrier:
We don't see the usage of keeping `extendedPropertyDefinition`.

We would like to hear from you @sergehuber

**Pull request merged**